.travis.yml

```diff
@@ -1,10 +1,16 @@
 language: node_js
 
+env:
+  - HAPI_VERSION="19"
+  - HAPI_VERSION="20"
+
 node_js:
-  - "8"
-  - "10"
   - "12"
   - "14"
   - "node"
 
+install:
+  - "npm install"
+  - "npm install @hapi/hapi@$HAPI_VERSION"
+
 after_script: "npm run coveralls"
```

Comment:
You read my mind— this all looks great.  I do want to start adding this snippet to our travis configs as well, though.
```suggestion
after_script: "npm run coveralls"

jobs:
  fast_finish: true
  allow_failures:
    - node_js: "node"
```

Reply:
Alright, I started an issue on https://github.com/hapipal/discuss/issues/5 as well